packages/api-plugin-fulfillment-method-pickup-store/src/startup.js

```diff
@@ -0,0 +1,22 @@
+import ReactionError from "@reactioncommerce/reaction-error";
+import checkAndCreateFulfillmentMethod from "./util/checkAndCreateFulfillmentMethod";
+/**
+ * @summary Called on startup to create the root entry of this fulfillment type in Fulfillment collection
+ * @param {Object} context Startup context
+ * @param {Object} context.collections Map of MongoDB collections
```

Comment:
This doesn't seem to make any sense here

Reply:
removed